src/components/UserManagedList.js

```diff
@@ -167,36 +168,46 @@ export default class UserManagedList extends SystemManagedList {
   }
 
   buildSaveSpec(list) {
-    return list.reduce((result, component) => {
-      result.push(EntitySpec.fromCoordinates(component))
-      return result
-    }, [])
+    return list.map(component => EntitySpec.fromCoordinates(component))
   }
 
-  doSave() {
+  async doSave() {
     const { components } = this.props
     const spec = this.buildSaveSpec(components.list)
-    this.saveSpec(spec)
-    this.setState({ showSavePopup: false, fileName: null })
+    await this.saveSpec(spec)
+    this.setState({ showSavePopup: false })
   }
 
   async saveSpec(spec) {
     const { dispatch } = this.props
+    const { options } = this.state
     try {
+      if (this.state.saveType === 'notice') return this.saveNotices(spec, options)
       const fileObject = { filter: this.state.activeFilters, sortBy: this.state.activeSort, coordinates: spec }
-      if (this.state.saveType === 'gist') await this.createGist(this.state.fileName, fileObject)
-      else {
-        const file = new File([JSON.stringify(fileObject, null, 2)], `${this.state.fileName}.json`)
-        saveAs(file)
-      }
+      if (this.state.saveType === 'gist') await this.saveGist(options.filename, fileObject)
+      else saveAs(new File([JSON.stringify(fileObject, null, 2)], `${options.filename}.json`))
     } catch (error) {
       if (error.status === 404)
         return uiWarning(dispatch, "Could not create Gist. Likely you've not given us permission")
       uiWarning(dispatch, error.message)
     }
   }
 
-  async createGist(name, content) {
+  async saveNotices(coordinates, options) {
+    const { token, dispatch } = this.props
+    uiInfo(dispatch, `Creating notice file for ${coordinates.length} coordinates...`)
+    const list = coordinates.map(entry => entry.toString())
+    const notices = await getNotices(token, list, options.renderer, options.options)
+    const { summary, content } = notices
+    const message = `Created Notices file with ${summary.total} entries, ${summary.missing} missing entries, and ${
+      summary.warnings.noLicense.length
+    } missing licenses`
```

Comment:
let's add a link in this toast to fix the ones with missing licenses.
This will filter the list to 'absence of license' so they can be curated